contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -13,7 +13,7 @@ As techdocs HTML pages load assets without an Authorization header the code belo
 
 import cookieParser from 'cookie-parser';
 import { Request, Response, NextFunction } from 'express';
-import { JWT } from 'jose';
+import jwtDecoder from 'jwt-decode';
```

Comment:
We're mostly sticking to `jose`, did you have any specific reason for the switch?

Reply:
Silly mistake on my end - it works fine.

Reply:
@Rugvip I've actually had issues with jose on my Windows machine. I'm getting an undefined JWT object - not sure why, didn't explore it further.

Reply:
Would guess it's because of a versioning issue where you end up getting v2 or v3 picked up rather than v1